### PR TITLE
add updates based on new server releases

### PR DESCRIPTION
**Pull request by gcolliso**

### CODE REVIEW

File: nats_server/configuration.md [hunk 1]
```diff
@@ -101,26 +101,29 @@ authorization: {
 
 | Property | Description |
 | :------  | :---- |
-| [`authorization`](auth_intro.md) | configuration map for client authentication/authorization |
-| [`cluster`](cluster_config.md) | configuration map for clustering configuration |
-| `debug` | if `true` enable debug log messages |
+| [`authorization`](auth_intro.md) | Configuration map for client authentication/authorization |
+| [`cluster`](cluster_config.md) | Configuration map for clustering configuration |
+| `connectionerrorreportattempts` | Number of attempts at which a repeated failed route, gateway or leaf node connection is reported. Default is 1 hour |
```
Comment (ColinSullivan1) [inline nats_server/configuration.md]: `connect_error_reports`, there's also `reconnect_error_reports`

File: nats_server/configuration.md [hunk 2]
```diff
@@ -101,26 +101,29 @@ authorization: {
...
 | [`leafnode`](/leafnodes/leafnode_conf.md) | Leafnode configuration map |
-| `listen`   | host/port for client connections |
+| `listen`   | Host/port for client connections |
 | `max_connections` | Maximum number of active client connections |
 | `max_control_line` | Maximum length of a protocol line (including subject length) |
 | `max_payload` | Maximum number of bytes in a message payload |
 | `max_pending` | Maximum number of bytes buffered for a connection |
 | `max_subscriptions` | Maximum numbers of subscriptions for a client connection |
-| [`operator`](/nats_tools/nsc/nsc.md#nats-server-configuration) | path to an operator JWT |
+| `maxtracedmsglen` | Set a limit to the trace of the payload of a message |
```
Comment (ColinSullivan1) [inline nats_server/configuration.md]: `max_traced_msg_len`

File: nats_server/configuration.md [hunk 3]
```diff
@@ -101,26 +101,29 @@ authorization: {
...
-| `listen`   | host/port for client connections |
+| `listen`   | Host/port for client connections |
 | `max_connections` | Maximum number of active client connections |
 | `max_control_line` | Maximum length of a protocol line (including subject length) |
 | `max_payload` | Maximum number of bytes in a message payload |
 | `max_pending` | Maximum number of bytes buffered for a connection |
 | `max_subscriptions` | Maximum numbers of subscriptions for a client connection |
-| [`operator`](/nats_tools/nsc/nsc.md#nats-server-configuration) | path to an operator JWT |
+| `maxtracedmsglen` | Set a limit to the trace of the payload of a message | 
+| `nosublistcache` | Disable sublist cache globally for accounts.
```
Comment (ColinSullivan1) [inline nats_server/configuration.md]: `disable_sublist_cache`

**Comment by ColinSullivan1** [comment]
looks like some underscores were dropped along the way...

File: nats_server/configuration.md (hunk 3 above)
Comment (gcolliso) [reply nats_server/configuration.md]: does this mean add `disable_sublist_cache` or replace with `disable_sublist_cache`?

Comment (ColinSullivan1) [reply nats_server/configuration.md]: imo let's go with one, replace it with disable_sublist_cache.  Reads better to me...

**Pull request merged**